src/main/java/ch/njol/skript/expressions/ExprSwimming.java

```diff
@@ -0,0 +1,75 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Swimming")
+@Description("The swimming state of living entities, present in 1.13+")
+@Examples("set the swimming state of target entity to true")
+@Since("INSERT VERSION")
+public class ExprSwimming extends SimplePropertyExpression<LivingEntity, Boolean> {
+
+	static {
+		if (Skript.methodExists(LivingEntity.class, "isSwimming"))
+			register(ExprSwimming.class, Boolean.class, "swimming [state]", "livingentities");
+	}
+	
+	@Override
+	public Class<Boolean> getReturnType() {
```

Comment:
convert
acceptChange
change
getReturnType
getPropertyName

Reply:
Are you referring to the order or what? That's the wrong order if you are.

Reply:
This order was suggested by @Nicofisi in Discord but i think getPropertyName should be at the end, and we used this recently

Reply:
@Blueyescat I've just checked 10 random `SimplePropertyExpression`s and 9 of them had getPropertyName before getReturnType, although but I'm open to discussion of course 🍿

Reply:
If toString is at the end and if getPropertyName is actually toString no need to discussion 😜

Reply:
Quoting me from Discord:
> idk if it's worth editing most of the property expressions just for that
as i said, i checked random 10 ones and 9 of them had getPropertyName before getReturnType
if someone was editing most of the expressions for some other reason though, like to improve the code in general, use lambdas, fix all methods orders etc though
then sure

Anyway, please follow what @Blueyescat wrote @TheLimeGlass, or change that one thing if you hate it too in the order I guess lol

Can do it for you when merging if it'd take too long to open the IDE only to edit just that on your pc or something